tests/components/backup/test_websocket.py

```diff
@@ -67,9 +88,169 @@ async def test_generate(
         "homeassistant.components.backup.websocket.BackupManager.generate_backup",
         return_value=TEST_BACKUP,
     ):
-        await client.send_json({"id": 1, "type": "backup/generate"})
-        msg = await client.receive_json()
+        await client.send_json_auto_id({"type": "backup/generate"})
+        assert snapshot == await client.receive_json()
+
+
+@pytest.mark.parametrize(
+    ("with_hassio"),
+    (
+        pytest.param(True, id="with_hassio"),
+        pytest.param(False, id="without_hassio"),
+    ),
+)
+async def test_backup_end_as_supervisor(
+    hass: HomeAssistant,
+    hass_ws_client: WebSocketGenerator,
+    snapshot: SnapshotAssertion,
+    hass_supervisor_access_token: str,
+    with_hassio: bool,
+) -> None:
+    """Test handling of post backup actions from a WS command."""
+    await setup_backup_integration(hass, with_hassio=with_hassio)
+
+    client = await hass_ws_client(hass, hass_supervisor_access_token)
```

Comment:
Can we make the access token a test parameter?

Reply:
Sync fixtures seem to be fine with using `request.getfixturevalue`, but because of <https://github.com/pytest-dev/pytest-asyncio/issues/112> that's not working with async fixtures.